qdrant-landing/content/documentation/integrations/dspy.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Stanford DSPy
+weight: 1500
+---
+
+# Stanford DSPy
+
+[DSPy](https://github.com/stanfordnlp/dspy) is the framework for solving advanced tasks with language models (LMs) and retrieval models (RMs). It unifies techniques for prompting and fine-tuning LMs — and approaches for reasoning, self-improvement, and augmentation with retrieval and tools.
+
+- Provides composable and declarative modules for instructing LMs in a familiar Pythonic syntax.
+
+- Introduces an automatic compiler that teaches LMs how to conduct the declarative steps in your program.
+
+Qdrant can be used as a retrieval mechanism in the DSPy flow.
+
+## Installation
+
+For the Qdrant retrieval integration, include `dspy-ai` with the `qdrant` extra:
+```bash
+pip install dspy-ai[qdrant]
+```
+
+## Usage
+
+We can configure `DSPy` settings to use the Qdrant retriever model like so:
+```python
+import dspy
+from dspy.retrieve.qdrant_rm import QdrantRM
+
+from qdrant_client import QdrantClient
+
+turbo = dspy.OpenAI(model='gpt-3.5-turbo')
+qdrant_client = Qdrant()    # Defaults to a local instance at http://localhost:6333/
```

Comment:
Shouldn't it be `QdrantClient`?

Reply:
https://github.com/qdrant/landing_page/pull/390/commits/5475fa800ee3badab782b970f39026729cc686ab.
Fixed.